signinapp/templates/profile.html.jinja2

```diff
@@ -16,18 +16,21 @@
             <div class="border border-5" id="qrPlaceholder"></div>
           </div>
           <h1 class="d-flex justify-content-center pt-3">{{ user.name }}</h1>
-          <h2 class="fw-light text-dark d-flex justify-content-center pb-3">{{ user.subteam.name }}</h2>
+          <h2 class="fw-light text-dark d-flex justify-content-center">{{ user.subteam.name }}</h2>
+          {% if user.student_user_data %}
+          <h2 class="fw-light text-dark d-flex justify-content-center pb-3">{{ user.student_user_data.display_grade }}</h2>
+          {% endif %}
         </div>
       </div>
       <div class="col-xl-8 px-4">
-        {% if user.guardians %}
+        {% if user.student_user_data.guardians %}
```

Comment:
Does this actually work? I would expect that it triggers an attribute error if user data is none, unless Jinja hides it

Reply:
This *does* seem to work surprisingly. Is it worth making it?
```suggestion
        {% if user.student_user_data and user.student_user_data.guardians %}
```